docs/developers/bitesize/ci/travis_centaur.md

```diff
@@ -0,0 +1,75 @@
+## Travis builds by category of user
+
+| Backend | Any User | Committers |
```

Comment:
What's a committer? Do you mean what Github would mark as a collaborator? I thought it was more to do with "is the branch in the broadinstitute/cromwell repo" rather than the user profile of who was making the PR?

Reply:
Will try to make up another short description... 🤔